model/src/main/kotlin/OrtResult.kt

```diff
@@ -378,13 +378,13 @@ data class OrtResult(
         }
 
     /**
-     * Return all [AdvisorResultContainer]s contained in this [OrtResult] or only the non-excluded ones if
-     * [omitExcluded] is true.
+     * Return all [AdvisorResult]s contained in this [OrtResult] or only the non-excluded ones if [omitExcluded] is
+     * true.
      */
     @JsonIgnore
-    fun getAdvisorResultContainers(omitExcluded: Boolean = false): Set<AdvisorResultContainer> =
-        advisor?.results?.advisorResults.orEmpty().filterTo(mutableSetOf()) { result ->
-            !omitExcluded || !isExcluded(result.id)
+    fun getAdvisorResults(omitExcluded: Boolean = false): Map<Identifier, List<AdvisorResult>> =
```

Comment:
Note that `reporter/src/main/resources/templates/asciidoc/vulnerability_report.ftl` still has a call to `ortResult.getAdvisorResultContainers()`.

Reply:
Thanks for the hint, done. Very unfortunate that there is no test case for this.